commons/monitor/application/application_device.avsc

```diff
@@ -0,0 +1,19 @@
+{
+  "namespace": "org.radarcns.monitor.application",
+  "type": "record",
+  "name": "ApplicationDevice",
+  "doc": "Device information where the application is running.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Device timestamp as time since the Unix Epoch (s)." },
+    { "name": "operatingSystem", "type": {
+      "name": "OperatingSystem",
+      "type": "enum",
+      "symbols": ["ANDROID", "IOS", "MACOS", "WINDOWS", "LINUX", "BSD", "OTHER", "UNKNOWN"],
+      "doc": "Operating system types."
+      }, "doc": "Operating system that the application is running on.", "default": "UNKNOWN"},
+    { "name": "releaseVersion", "type": ["null", "string"], "doc": "Version as readable string. Null if unknown.", "default": null },
+    { "name": "releaseVersionCode", "type": ["null", "int"], "doc": "Unique version number for the operating system. Null if unknown.", "default": null },
+    { "name": "manufacturer", "type": ["null", "string"], "doc": "Manufacturer of the device. Null if unknown.", "default": null },
+    { "name": "model", "type": ["null", "string"], "doc": "Model of the device. Null if unknown.", "default": null }
+  ]
```

Comment:
Is it possible to retrieve the build number ? It could be useful to debug something in case of a vendor OTA update.